bia_agent/mifa2pagetab.py

```diff
@@ -71,13 +71,15 @@ def mifa_container_to_pagetab(container: REMBIContainer, accession_id: Optional[
     def rembi_objects_to_pagetab_sections(conversion_func, objects_dict):
 
         sections = [
-            conversion_func(object, object_id)
+            conversion_func(object, title=object_id)
```

Comment:
Should the title keyword be added to the other <X>_to_pagetab_sections functions to keep them all consistent?

Reply:
Good point, I'll do that.